src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Figure implements AreaCalculator, Drawable {
```

Comment:
When you made some methods `abstract` your class should be defined also as a `abstract`